src/backend/egl/display.rs

```diff
@@ -906,6 +906,10 @@ impl EGLBufferReader {
         &self,
         buffer: &WlBuffer,
     ) -> ::std::result::Result<EGLBuffer, BufferAccessError> {
+        if !buffer.is_alive() {
+            return Err(BufferAccessError::NotManaged(EGLError::BadAccess));
```

Comment:
@Drakulix @i509VCB not sure if this is the best suited error, maybe something like `BufferAccessError::Destroyed` would be a better fit?

Reply:
`BufferAccessError::Destroyed` sounds better than `BufferAccessError::NotManaged` yeah

Reply:
Added `BufferAccessError:Destroyed` and updated `egl_buffer_contents`